packages/headless/src/database/mongodb/cart/cart.model.ts

```diff
@@ -0,0 +1,35 @@
+import { string } from 'joi';
+import { model, Schema } from 'mongoose';
+import { Cart } from 'src/entity/cart';
+
+const CartSchema = new Schema<Cart>({
+  user: {
+    type: String,
+    required: true,
+  },
+  items: [
+    {
+      product: {
+        type: String,
+        required: true,
+      },
+      quantity: {
+        type: Number,
+        required: true,
+        default: 1,
+      },
+      _id: false,
+    },
+  ],
+  createdOn: {
+    type: Date,
+    default: new Date(),
+  },
+  updatedOn: {
+    type: Date,
+    default: new Date(),
+  },
```

Comment:
recommended to use mongoose createdAt, updatedAt value. you can check mongoose timestamp flag in mongoose model constructor function

Reply:
done